manifests/operator.yaml

```diff
@@ -75,32 +75,37 @@ rules:
 - apiGroups: [""]
   resources:
   - configmaps
-  verbs: ["create", "patch", "update"]
+  verbs: ["get", "list", "watch", "create", "patch", "update"]
 - apiGroups: ["apps"]
   resources:
   - daemonsets
-  verbs: ["create", "delete", "patch", "update"]
-- apiGroups: ["apps"]
-  resources:
   - deployments
-  verbs: ["create", "delete", "patch", "update"]
+  verbs: ["get", "list", "watch", "delete", "patch", "update"]
```

Comment:
are there particular deployments that you want to delete, patch and update?

Reply:
Just within the operator's own namespace (`gmp-system`). We delete, patch and update [as part of a cleanup](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/pkg/operator/operator.go#L309).

Reply:
right but do we know before hand the name of the deployment? If so we can restrict this even more by using resourceNames.